Esri/arcgis-maps-sdk-swift-samples
# [New] Show device location using indoor positioning

## Description

This PR implements **`Show device location using indoor positioning`** in `Map` category.

- `swift/issues/5653` (internal test data in the issue)
- `common-samples/pull/3939`

## How To Test

Load map and wait for campus to load. You should see your location relative to the layout of the office.

Please note:

- The sample data doesn't work in Building M due to no pathway table was surveyed, and not in Building L either maybe due to dead batteries. To see it in action, use the internal data in the issue linked above.
- Both the `indoorPositioningDefinition` and the manual `ips_positioning` creation of the LDS should work, so make sure you test both condition branches

## Screenshots

![351891359-cf184145-a083-48f0-a5a0-3dbc6ca7c106](https://github.com/user-attachments/assets/d3bb7fbc-2f13-4702-a51b-ba44a43e6b67)
